### ISSUE TITLE
Pagination

### ISSUE THREAD

**Issue by shreyasb07** [yurydelendik/pdfjs-react#4]
Hi Yury,

This is an amazing project! I've seen many examples, but have found your's to be very helpful! However, I was wondering if we could have a pagination option instead of viewing all the pages one below the other? Probably by using some state management? Looking forward to hearing from you soon!

**Comment by yurydelendik** [comment]
@shreyasb07 yeah, you will need some additional state management. The final solution is a mix of:
- https://github.com/mozilla/pdf.js/blob/master/examples/components/pageviewer.js
- https://jsfiddle.net/pdfjs/wagvs9Lf/?utm_source=website&utm_medium=embed&utm_campaign=wagvs9Lf
- and this example

**Comment by shreyasb07** [comment]
Hi Yury @yurydelendik ,

Thank you for your prompt reply! I tried referring to the above 2 links you've provided. However, I could still see all of the pages one below the other! I'd really appreciate your help on this!
